example/ios/Podfile.lock

```diff
@@ -0,0 +1,51 @@
+PODS:
+  - GoogleMaps (2.0.1):
+    - GoogleMaps/Maps (= 2.0.1)
+  - GoogleMaps/Base (2.0.1)
+  - GoogleMaps/Maps (2.0.1):
+    - GoogleMaps/Base (= 2.0.1)
+  - React/Core (0.32.1):
```

Comment:
Sure. I will change the version in `examples/package.json` from `^0.32.0` --> `0.32.0` so this doesn't happen.